src/game/start.py

```diff
@@ -5,133 +5,131 @@
 from .data import Piece, GameTurn, GameState, PIECES_NUMBER
 
 
-def start_game():
-    print("Welcome to Quarto-Py")
+class UIRender:
```

Comment:
I'm not sure that the name is good here. There's a lot of logic that is not part of "UI" like "start_game" ...
You should split this class into another IMO (one for interface and another one for game management)

Moreover, the name of the file "start.py" should mirror the class name ideally